artmaker/src/main/java/io/artmaker/composables/ArtMakerDrawScreen.kt

```diff
@@ -191,4 +209,46 @@ internal fun ArtMakerDrawScreen(
             }
         },
     )
+
+    if (shouldShowStylusDialog && state.canShowStylusDialog) {
+        if (stylusDialogType.isEmpty()) return
+        val type = StylusDialogType.valueOf(stylusDialogType)
+        val dialogInfo = when (type) {
+            StylusDialogType.ENABLE_STYLUS_ONLY -> stringResource(R.string.stylus_input_detected_title) to stringResource(R.string.styluse_input_detected_message)
+            StylusDialogType.DISABLE_STYLUS_ONLY -> stringResource(R.string.non_stylus_input_detected_title) to stringResource(R.string.non_stylus_input_detected_message)
+        }
+
+        AlertDialog(
```

Comment:
This is a nice to have for palm rejection. When drawing with a stylus and the user has set use only stylus then try drawing with the finger, we show a dialog to tell them why they cannot draw. 

If they have started drawing with a stylus but have not enabled stylus only input, then we also show them this dialog to tell them that a better experience can be had by enabling stylus only input. 


There is an option for hiding this dialog forever.

Reply:
This is good, We can have one button for this and remove the `Okay` button.

Reply:
Thus enhancing the User Experience (UX). Nice...